.github/workflows/CI.yml

```
@@ -33,17 +33,19 @@ jobs:
           ${{ runner.os }}-gradle-
 
-    - name: 4. Copy application.yml
+    - name: 4. Copy application-prod.yml
       env:
-        CREATE_SECRET: ${{ secrets.APPLICATION_YML }}
-        CREATE_SECRET_DIR: src/main/resources
-        CREATE_SECRET_DIR_FILE_NAME: application.yml
+        YML: ${{ secrets.APPLICATION_PROD_YML }}
+        YML_DIR: src/main/resources
+        YML_NAME: application-prod.yml
       run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:10: Double quote to prevent globbing and word splitting

(shellcheck)